src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -137,13 +143,15 @@ export class NetworkRequest {
     return Boolean(this.#request.info);
   }
 
-  #isBlockedInPhase(phase?: Network.InterceptPhase) {
-    return this.#networkStorage.requestBlockedBy(this, phase).size > 0;
+  #isBlockedByInPhase(phase: Network.InterceptPhase) {
+    return this.#networkStorage.requestBlockedBy(this, phase);
```

Comment:
let's also rename the method in storage to be something like getInterceptsForRequestPhase